src/amm/amm.requests.ts

```diff
@@ -106,13 +180,36 @@ export interface AddLiquidityResponse {
   token0: string;
   token1: string;
   fee: string;
-  tokenId: number;
+  tokenId: number; // COSMOS: poolId
   gasPrice: number;
   gasPriceToken: string;
   gasLimit: number;
-  gasCost: string;
+  gasCost: string; // gasUsed for Cosmos
+  gasWanted?: string;
   nonce: number;
   txHash: string | undefined;
+  poolAddress?: string; // Cosmos only
+  poolShares?: string; // Cosmos only
+  token0FinalAmount?: string; // Cosmos only
+  token1FinalAmount?: string; // Cosmos only
+}
+
+export interface CosmosAddLiquidityResponse {
```

Comment:
Similarly please modify the existing addLiquidity response and request interfaces